pkg/operator/reports_test.go

```diff
@@ -92,3 +96,86 @@ func TestGetNextReportPeriod(t *testing.T) {
 		})
 	}
 }
+
+func TestReportRunningCondition(t *testing.T) {
```

Comment:
This is testing `isReportRunning` so the test name should be `TestIsReportRunning`